packages/react-ui-core/src/ToggleButton/__tests__/ToggleButton-test.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import ThemedToggleButton from '../ToggleButton'
+import theme from './mocks/theme'
+
+const ToggleButton = ThemedToggleButton.WrappedComponent
+
+describe('ToggleButton', () => {
+  it('renders', () => {
```

Comment:
`it('renders')` really doesn't tell me anything. They all render or fail.  How about testing for one specific thing here like children or the base class?

Reply:
Fixed